sycl/doc/extensions/Matrix/dpcpp-joint-matrix.asciidoc

```diff
@@ -60,17 +60,8 @@ We introduce a new class called `joint_matrix`. The user needs to specify the ty
 
 //`joint_matrix` is distributed among an execution unit. In practice this can be one work-item, the work-items in a sub-group, or the work-items in a work-group.
 
-#### Memory Scope
-In this experimental API version, we used the terminology of `joint_matrix` instead of plain `matrix` to emphasis that the matrix is shared among a group of work items and is not private to each work item. The memory scope is added as an additional template parameter and is also part of the constructor arguments.
-
-	
-#### Shape
-The same class `joint_matrix` should handle both cases where sizes are constant (GPU case) and when sizes are variables (CPU case). Note that a AMX 2d tile register permits sizes up to 1024 (16rowsx64cols) bytes. The ability to define only one interface for both makes it possible to give the user a way to make use of the flexibility introduced by the CPU but at the same time save resources on the GPU. We use `sycl::dynamic_extent`  to differentiate between static and dynamic sizes.
-	
-#### Layout
-Layout is necessary on the type to be able to calculate the physical offset if the user needs to access a single entry for some purpose. Besides row major and column major layout, `matrix_layout` is flexible enough to introduce customed layouts such as symmetric or tiled layouts.
 
-This results into the following description:
+//This results into the following description:
```

Comment:
yes that was the intent to not have them visible in the HTML version. But I agree, I will remove these comments because they will still be visible to people reading the ascii file